src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -51,58 +51,89 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
     * @return List<String> list of DataImport__c field api names
     */
     public static List<String> getDataImportFields(Id batchId) {
-        List<String> fieldNames = new List<String>();
+        return getDataImportFields(batchId, true);
+    }
+
+    /*******************************************************************************************************
+    * @description Provides field mapping configuration used by BGE
+    */
+    public static Map<SObjectField, BDI_TargetFields> targetFieldsBySourceField = 
+        BDI_DataImportService.getDefaultMappingService().getTargetFieldsBySourceField();
+
+    /*******************************************************************************************************
+    * @description returns a list of DataImport__c fields the Batch Gift Entry UI needs in SOQL
+    * @param batchId the batch for which to get all fields for soql
+    * @param includeRelationshipFields whether to include relationship fields in the returned list
+    * @return List<String> list of DataImport__c field api names
+    */
+    public static List<String> getDataImportFields(Id batchId, Boolean includeRelationshipFields) {
```

Comment:
Can we move this second implementation of "getDataImportFields" up directly below the first one? Or move the first one down next to here?